backend/entities/Models/PropertyModel.cs

```diff
@@ -0,0 +1,206 @@
+using System;
+using System.Linq;
+using System.Security.Claims;
+using NetTopologySuite.Geometries;
+using Pims.Core.Extensions;
+
+namespace Pims.Dal.Entities.Models
```

Comment:
A handful of new models that the DAL can return.  These specific ones are used to remove information from properties that are not owned by the user's agency or sub-agencies.